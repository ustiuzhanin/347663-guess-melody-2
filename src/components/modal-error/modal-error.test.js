import React from "react";
import renderer from "react-test-renderer";
import ModalError from "./modal-error.jsx";

test(`ModalError's Snapshot`, () => {
  const tree = renderer
    .create(
        <ModalError
          errorMessage={{
            status: 404,
            data: {
              error: `Not Found`
            }
          }}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
