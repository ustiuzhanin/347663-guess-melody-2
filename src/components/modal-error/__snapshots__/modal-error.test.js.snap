// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalError's Snapshot 1`] = `
<section
  className="modal"
>
  <h2
    className="modal__title"
  >
    An error occurred!
  </h2>
  <p
    className="modal__text"
  >
    Status: 404
        Not Found
    <br />
    Please reload the page
  </p>
</section>
`;
